### DISCUSSION TITLE
D-Link DWA-131 working on macOS 12.3 Monterey

### DISCUSSION THREAD

**Discussion by DarthDensus** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#215] [General]
The above mentioned adapter works on Monterey.

**Comment by chris1111** [comment]
Thanks [Done](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/commit/a4975944bd4f456f1fd8fc5c3631fa2ab1b77d18)
